src/Compilers/CSharp/Portable/Lowering/LambdaRewriter/LambdaRewriter.Analysis.cs

```diff
@@ -105,7 +100,10 @@ internal sealed partial class Analysis : BoundTreeWalkerWithStackGuardWithoutRec
             /// </summary>
             public Dictionary<MethodSymbol, BoundNode> LambdaScopes;
 
-            private Scope _scopeTree;
+            /// <summary>
+            /// The root of the scope tree for this method.
+            /// </summary>
+            public Scope ScopeTree { get; private set; }
```

Comment:
Delete the `private set`. It's unnecessary since you only modify the value in the ctor. Makes it clear this is a `readonly` property.